### DISCUSSION TITLE
how to link 2 custom field

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16877] [How to] [answered]
I created a custom field PRODUCT SPEC in product, and a custom field PRODUCT SPEC in Quotes [advanced block],  want to choose the product name, then the PRODUCT SPEC show up like other field price....please advise how to make it... appreciate.

**Comment by newminder** [comment]
I tried reference field, but does not work

**Comment by mariuszkrzaczkowski** [comment]
1. question or error?
- question is discussion
- bug - see https://github.com/YetiForceCompany/YetiForceCRM/issues/new

3. why did you specify "Docker"? the question is about CRM

**Comment by newminder** [comment, marked as answer]
should be a question, sorry, i'm a rookie here.   
when i choose the item from the product, the product name, uni, price...all are fine, except the custom field PRODUCT SPEC is blank

**Reply by newminder** [reply]
already solved, map the 2 fields in db
